README.md

```diff
@@ -201,6 +201,100 @@ echo $response->getRecommendation()->getStatus(); // OK
 $recommendations = $response->getRecommendation()->getData();
 ```
 
+#### Recommendation response format
+
+*New in Matej v7.39*
```

Comment:
This could be omitted, if we merge this only after 7.39 is on production :)

Reply:
Also, as we discussed personally, we may target this to v2.0 of the client, and transparently declare change of the Recommendation format (from array of ids to array of objects with multiple properties) as a BC break.

Thoughts from others? @jirinovak @tenerd etc. ?